installer/templates/phx_umbrella/apps/app_name_web/config/runtime.exs

```diff
@@ -15,7 +15,7 @@ config :<%= @web_app_name %>, <%= @endpoint_module %>,
     # Enable IPv6 and bind on all interfaces.
     # Set it to  {0, 0, 0, 0, 0, 0, 0, 1} for local network only access.
     ip: {0, 0, 0, 0, 0, 0, 0, 0},
-    port: String.to_integer(System.get_env("PORT") || "4000")
+    port: port
```

Comment:
I noticed this file gets injected into `umbrella/config/runtime.exs`. Unlike `phx_single` the template files are not 1:1 with the final config files in a generated umbrella project.

This has led to something like #6073, even though I saw code that surgically remove "import Config" in some places :)

That's to say I hope this change is okay, as it relies on `port` being defined elsewhere.